services/src/datasets/external/gbif.rs

```diff
@@ -2069,6 +2073,7 @@ mod tests {
                     .iter()
                     .cloned()
                     .collect(),
+                    CacheHint::default(), // TODO: make configurable in data provider(?)
```

Comment:
sounds reasonable

Reply:
ok, but let's first agree on how we model ttl (seconds or enum)

Reply:
5c04d280f009ec129d8b8fb5a92e3fc1ca61c65b